### ISSUE TITLE
Wish List/Feature Request: Image Caption

### ISSUE THREAD

**Issue by jdhitsolutions** [OpenLiveWriter/OpenLiveWriter#230]
I would love to have a menu option to insert an image caption. Right now I'm experimenting with manually inserting the HTML tags around the image.

**Comment by ScottIsAFool** [comment]
This sounds like a good idea for a plugin

**Comment by Dinchy87** [comment]
Yep, needed this also and added always a line under the picture with text set to align in the center.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
